src/gdcdictionary/schemas/treatment.yaml

```diff
@@ -6945,7 +6970,6 @@ properties:
       - Colon
       - Colon - Mucosa Only
       - Common Duct
-      - Conjunctiva
```

Comment:
I can't find the direction to delete this enum in [DICT-516](https://gdc-ctds.atlassian.net/browse/DICT-516)

Reply:
Fixed